addons/point_of_sale/static/src/app/components/order_tabs/order_tabs.js

```diff
@@ -36,7 +36,7 @@ export class OrderTabs extends Component {
         if (previousOrderScreen?.name === "PaymentScreen") {
             props.orderUuid = order.uuid;
         }
-
+        this.pos.get_order().deselect_orderline();
```

Comment:
I'm wondering something, since no orderline needs to be selected when arriving on product_screen, can tyou maybe just put a condition in `showScreen` method like:

```
if (screen === "ProductScreen") {
    this.pos.get_order().deselect_orderline();
}
```

Like that you don't need to put it everywhere. Just at one place and you ensure that it pass everytime.

Reply:
Yes it's better, I've made the changes, and I'm waiting for run bot to see if there is failing tests